lib/nostrum/shard/session.ex

```diff
@@ -41,7 +41,7 @@ defmodule Nostrum.Shard.Session do
   end
 
   def start_link([gateway, shard_num]) do
-    GenServer.start_link(__MODULE__, [gateway, shard_num], spawn_opt: [Util.fullsweep_after()])
+    GenServer.start_link(__MODULE__, [gateway, shard_num])
```

Comment:
I'm pretty skeptical of this change. I took a look at the GenServer start options [here](https://hexdocs.pm/elixir/1.9.1/GenServer.html#t:option/0) and nothing immediately looks amiss. I think we should look into this more as this is effectively removing functionality.

Reply:
As i said, i havent looked at this pr for a while, but iirc there was some error that kept spawn_opt from being accepted as valid. I can double check this change in a bit tho, since the docs do suggest that it should work

Reply:
ah, found it; just removing the list from around the call to `Util.fullsweep_after` is enough